cypress/e2e/landing-spec.cy.js

```diff
@@ -12,21 +12,12 @@ describe('Landing Page', () => {
     cy.visit('/')
   })
 
-/* Test banner contents 
-  - Test for logo
-  - Test for search bar
-  - Test for radio buttons
-  - Test for search button
-  - Test for welcome message (no user login)
-  - Test for "Home | Login | Favorites"
```

Comment:
Love seeing this! Always remove commented out(dead code) before pushing up code.